content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Scale Sensu Go with Enterprise Datastore"
+linkTitle: "Routing Alerts with Filters"
+description: ""
+weight: 39
+version: "5.10"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.10:
+    parent: guides
+---
+
+Sensu Go's licensed-tier datastore feature enables scaling your monitoring to many thousands of events per second.
+
+- [Why use the Enterprise Datastore?](#why-enterprise-datastore)
+- [Prerequisites](#prerequisites)
+- [Configuring Postgres](#configuring-postgres-for-sensu)
+- [Configuring Sensu](#configuring-sensu-for-postgres)
+- [Reverting to built-in datastore](#reverting-to-builtin-datastore)
+
+## Why use the Enterprise Datastore? {#why-enterprise-datastore}
+
+For each unique entity/check pair, Sensu records the latest event object in it's datastore. By default, the embedded etcd datastore is used for event storage. The embedded etcd datastore makes it easy to get started with Sensu, but as the number of entities and checks in your Sensu implementation grows, so too does the rate of events being written to the datastore. In a clustered deployment of etcd, whether embedded or external to Sensu, each event received by a member of the cluster must be replicated to other members, increasing network and disk IO utilization. 
+
+Our team has documented configuration and testing of Sensu running on bare metal infrastructure in the [sensu/sensu-perf][2] project. Under this configuration they were able to comfortably handle 12,000 Sensu Agent connections (and their keepalives) and process over 8,500 events per second. 
+
+This rate of events should be more than sufficient for many installations, but also assumes an ideal scenario where Sensu backend nodes use direct-attached, dedicated NVMe storage and are connected to a dedicated LAN. Deployments on public cloud providers are not likely to achieve similar results due to sharing both disk and network bandwidth with other tenants. Following cloud provider recommended practices can also become a factor, as may operators are inclined to deploy a cluster across multiple availability zones. In such a deployment cluster communication happens over shared WAN links which are subject to uncontrolled variability in throughput and latency.
+
+Using the Enterprise Datastore can help operators achieve much higher rates of event processing, and minimize the replication communication between etcd peers. The sensu-perf test environment comfortably handles 40,000 Sensu Agent connections (and their keepalives) and processes over 36,000 events per second under ideal conditions. 
+
+## Prerequisites
+
+* Database server running Postgres 9.5 or later
+* Postgres database, or administrative access to create one
+* Postgres user with permissions to the database, or administrative access to create one
```

Comment:
```suggestion
* Postgres user with permissions to the database (or administrative access to create such a user)
```

Reply:
This suggestion assumes that "one" refers to the required Postgres user. Or did you mean the permissions?